test/options-manager.js

```diff
@@ -645,7 +645,7 @@ test('mergeWithFileConfig: auto generated ts config extends found ts config if f
 test('mergeWithFileConfig: used found ts config if file is covered', async t => {
 	const cwd = path.resolve('fixtures', 'typescript', 'extends-config');
 	const filePath = path.resolve(cwd, 'foo.ts');
-	const expectedConfigPath = path.resolve(cwd, 'tsconfig.json');
+	const expectedConfigPath = slash(path.resolve(cwd, 'tsconfig.json'));
```

Comment:
In the other tests, you use `slash` on the `options.tsConfigPath`.

Reply:
Right, here no need for either.